provider/vpc/resource/security.py

```diff
@@ -3,30 +3,42 @@
 from typing import List
 
 from provider.vpc.command import VpcOptions, check_ipvpc_inpolicy
-from shared.common import *
+from shared.common import (
+    ResourceProvider,
+    Resource,
+    message_handler,
+    ResourceDigest,
+    ResourceEdge,
+    datetime_to_string,
+)
 from shared.error_handler import exception
 
 
-class IAMPOLICY(object):
-
+class IAMPOLICY(ResourceProvider):
     def __init__(self, vpc_options: VpcOptions):
+        """
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [One-line docstring should fit on one line with quotes (found 4) (D200)](https://app.codacy.com/gh/Cloud-Architects/cloud-discovery/pullRequest?prid=5655786)